include/micm/solver/jit_rosenbrock.hpp

```diff
@@ -113,7 +113,7 @@ namespace micm
       }
       else
       {
-        throw std::runtime_error("Failed to generate the alpha minus jacobia JIT function.");
+        throw std::system_error(make_error_code(MicmJitErrc::MissingJitFunction), "AlphaMinusJacobian");
```

Comment:
```suggestion
        throw std::system_error(make_error_code(MicmJitErrc::MissingJitFunction), std::source_location::current().function_name());
```

Reply:
that's a neat trick

Reply:
Oh, this requires the `#include <source_location>` header.

Reply:
pretty cool! done!